schemas/sppi/v2.json

```diff
@@ -0,0 +1,175 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "SPPI Survey SDS Schema",
+  "description": "Schema for the SPPI survey.",
+  "type": "object",
+  "properties": {
+    "schema_version": {
+      "const": "v2",
+      "description": "Version of the schema spec"
+    },
+    "survey_id": {
+      "type": "string",
+      "description": "The identifier for a particular survey",
+      "enum": ["061"]
+    },
+    "identifier": {
+      "type": "string",
+      "description": "The unique top-level identifier. For business surveys this is the reporting unit reference without the check letter appended.",
+      "minLength": 11,
+      "maxLength": 11,
+      "examples": ["34942807969"]
```

Comment:
Example number here and in supplier_number don't match the spreadsheet. Does that matter?

Reply:
Did you mean `items.service.identifier` and `items.service.item_number`? These were different compared to the spreadsheet. I've changed them now.